src/WikibaseRdfExtension.php

```diff
@@ -135,10 +135,13 @@ private static function getAllowedPredicates(): array {
 		return (array)MediaWikiServices::getInstance()->getMainConfig()->get( 'WikibaseRdfPredicates' );
 	}
 
-	public function newSaveMappingsUseCase( SaveMappingsPresenter $presenter ): SaveMappingsUseCase {
+	public function newSaveMappingsUseCase(
+		SaveMappingsPresenter $presenter,
+		Authority $authority
+	): SaveMappingsUseCase {
 		return new SaveMappingsUseCase(
 			$presenter,
-			$this->newMappingRepository( RequestContext::getMain()->getUser() ),
+			$this->newMappingRepository( $authority ),
```

Comment:
The changes in this class are because the `RequestContext` user isn't the one that the tests create.